.github/workflows/welcome-new-contributors.yml

```
@@ -0,0 +1,60 @@
+name: Welcome New Contributors
+
+on:
+  pull_request:
+    types: [ opened ]
+
+jobs:
+  post-welcome-message:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: bubkoo/welcome-action@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          FIRST_PR_COMMENT: >
+            Hi @{{ author }}! 👋
+
+
+            Thank you for your contribution to WordPress! 💖
+
+
+            It looks like this is your first pull request, so here are a few things to be aware of that may help you out.
+
+
+            **No one monitors this repository for new pull requests.** Pull requests **must** be attached to a Trac ticket to be considered for inclusion in WordPress Core. To attach a pull request to a Trac ticket, simply include the ticket's full URL in your pull request description.
```

```suggestion
            **No one monitors this repository for new pull requests.** Pull requests **must** be attached to a Trac ticket to be considered for inclusion in WordPress Core. To attach a pull request to a Trac ticket, please include the ticket's full URL in your pull request description.
```

No `simply`, please :)